API.md

```diff
@@ -0,0 +1,35 @@
+# sdc-uaa endpoints
+
+This page documents the sdc-uaa endpoints that can be hit.
+
+`/oauth/token`
+* POST request to this endpoint logs in a client, admin, or user.
+
+`/Groups`
+* GET request to this endpoint shows the results of a group search query.
+* POST request to this endpoint creates a new group
+
+`/Users`
+* GET request to this endpoint shows the results of a user search query.
+* POST request to this endpoint creates a new user.
+
+`/Groups/<group_id>/members`
+* POST request to this endpoint adds a user to a group.
+* `group_id` is the ID of the group.
+
+`/oauth/clients/{<client_id>/secret`
+* PUT request to this endpoint changes a client to a secret one.
```

Comment:
Can you give a bit more of a fuller explanation of what this means?  If it's documented well in the code then don't worry, otherwise it'd be helpful to understand what this is doing as I don't really get what 'changing a client to a secret one' really means